db/migrate/20220301170459_allow_bookings_placement_date_virtual_to_be_null.rb

```diff
@@ -0,0 +1,9 @@
+class AllowBookingsPlacementDateVirtualToBeNull < ActiveRecord::Migration[6.1]
+  def up
+    change_column :bookings_placement_dates, :virtual, :boolean, null: true
```

Comment:
This field is used to determine whether the virtual or in school booking confirmation is going to be sent. The app will fall back to `in school` for null values, but I think we shouldn't allow null values at all. It will create uncertainty in our reports and potentially more bugs in the future.